ui/job-view/Push.jsx

```diff
@@ -31,22 +31,33 @@ export default class Push extends React.Component {
       // props.push isn't actually immutable due to the way it hooks up to angular, therefore we
       // need to keep the previous value in the state.
       last_job_counts: job_counts ? { ...job_counts } : null,
+      hasBoundaryError: false,
+      boundaryError: '',
     };
   }
   componentDidMount() {
     this.showRunnableJobs = this.showRunnableJobs.bind(this);
     this.hideRunnableJobs = this.hideRunnableJobs.bind(this);
   }
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
This needs to be migrated to either ``getDerivedStateFromProps`` or some other mechanism.  But the code needed more re-work than I wanted to do in this PR, so I'm giving it the new name as a reminder.  [Bug 1451171](https://bugzilla.mozilla.org/show_bug.cgi?id=1451171)